packages/tools/generator-widget/generators/app/templates/packages/package_web.json.ejs

```diff
@@ -2,7 +2,7 @@
   "name": "<%- packageName %>",
   "widgetName": "<%- name %>",
   "version": "<%- version %>",
-  "description": "<%- description %>",
+  "description": "<%- description.replace(/["]/gi, '\\"') %>",
```

Comment:
Since we are embedding string literals in JSON output, a `JSON.stringify(...)`  would make more sense to me for all of these properties. (But then without the surrounding quotes, of course.)

Reply:
Don't forget to update the native templates, too :)